core/translate/expr.rs

```diff
@@ -1748,6 +1748,7 @@ pub fn translate_expr(
                             Ok(target_register)
                         }
                     },
+                    #[allow(unreachable_patterns)]
```

Comment:
I would remove `_ => unreachable!()` because having these are potential source of a bug when new variants are added to the enum.

Reply:
Yeah I believe I tried that when I implemented that feature. Unfortunately because that is the only variant, it will throw an error in this case when disabling the `uuid` feature.

![image](https://github.com/user-attachments/assets/9bf2d530-0829-4df6-90ef-3b9842a0799e)